libr/core/cmd_anal.c

```diff
@@ -196,15 +196,25 @@ static const char *help_detail_ae[] = {
 };
 
 static const char *help_msg_aea[] = {
-	"Examples:", "aea", " show regs used in a range",
-	"aea", " [ops]", "Show regs used in N instructions (all,read,{no,}written,memreads,memwrites)",
+	"Examples:", "aea", " show regs and memory accesses used in a range",
+	"aea", "  [ops]", "Show regs/memory accesses used in N instructions ",
 	"aea*", " [ops]", "Create mem.* flags for memory accesses",
 	"aeaf", "", "Show regs used in current function",
 	"aear", " [ops]", "Show regs read in N instructions",
 	"aeaw", " [ops]", "Show regs written in N instructions",
 	"aean", " [ops]", "Show regs not written in N instructions",
 	"aeaj", " [ops]", "Show aea output in JSON format",
-	"aeA", " [len]", "Show regs used in N bytes (subcommands are the same)",
+	"aeA", "  [len]", "Show regs used in N bytes (subcommands are the same)",
+	"Legend:", "", "",
+	"I", "", "input registers (read before being set)",
+	"A", "", "all regs accessed",
+	"R", "", "register values read",
+	"W", "", "registers written",
+	"NW", "", "= NW= not written",
```

Comment:
just "N" is the same, because NW appears in  one place like this, and N in another one. i think is better to just use one letter for everything

Reply:
Agreed